src/tests/ftest/recovery/pass_1.py

```diff
@@ -0,0 +1,416 @@
+
+        report_errors(test=self, errors=errors)
+
+    def test_lost_all_rdb(self):
+        """Remove rdb-pool from all mount point from all nodes. Now the pool can’t be
+        recovered, so checker should remove it from both MS and engine.
+
+        1. Create a pool.
+        2. Stop servers.
+        3. Remove /mnt/daos0/<pool_uuid>/rdb-pool from all ranks.
+        4. Start servers.
+        5. Enable and start the checker. It should remove pool from MS and engine.
+        6. Query the checker and verify the message.
+        7. Disable the checker.
+        8. Check that the pool does not appear with dmg pool list.
+        9. Verify that the pool directory was removed from the mount point.
+        10. Un-register the pool so that we don't try to destroy during tearDown.
+
+        Jira ID: DAOS-12067
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=recovery,pass_1
+        :avocado: tags=test_lost_all_rdb
+        """
+        # 1. Create a pool.
+        self.pool = self.get_pool()
+
+        # 2. Stop servers.
+        dmg_command = self.get_dmg_command()
+        dmg_command.system_stop()
+
+        # 3. Remove /mnt/daos/<pool_uuid>/rdb-pool from all ranks.
+        hosts = list(set(self.server_managers[0].ranks.values()))
+        nodeset_hosts = NodeSet.fromlist(hosts)
+        command = f"sudo rm /mnt/daos0/{self.pool.uuid.lower()}/rdb-pool"
+        pcmd(hosts=nodeset_hosts, command=command)
+
+        # 4. Start servers.
+        dmg_command.system_start()
+
+        # 5. Enable and start the checker.
+        dmg_command.check_enable()
+        # Calling dmg check start will automatically fix the issue with the default
+        # option, which is to discard the pool.
+        dmg_command.check_start()
+
+        # 6. Query the checker and verify the message.
+        query_msg = ""
+        for _ in range(8):
+            check_query_out = dmg_command.check_query()
+            if check_query_out["response"]["status"] == "COMPLETED":
+                if check_query_out["response"]["reports"]:
+                    query_msg = check_query_out["response"]["reports"][0]["msg"]
+                    break
+            time.sleep(5)
+
+        errors = []
+        if "corrupted pool without quorum" not in query_msg:
+            errors.append(
+                "Checker didn't detect orphan pool! msg = {}".format(query_msg))
+
+        # 7. Disable the checker.
+        dmg_command.check_disable()
+
+        # 8. Check that the pool does not appear with dmg pool list.
+        pools = dmg_command.get_pool_list_all()
+        if pools:
+            errors.append(f"Pool still exists after running checker! {pools}")
+
+        # 9. Verify that the pool directory was removed from the mount point.
+        errors = self.verify_pool_dir_removed(errors=errors)
+
+        # 10. Un-register the pool so that we don't try to destroy during tearDown.
+        self._cleanup_methods = []
```

Comment:
Similar to above. Recommend `pool.skip_cleanup()`

Reply:
Done